statistics/handle/update.go

```diff
@@ -492,9 +505,13 @@ func (h *Handle) DumpStatsDeltaToKV(mode dumpMode) error {
 		h.globalMap.data = deltaMap
 		h.globalMap.Unlock()
 	}()
+	// TODO: pass in do.InfoSchema() to DumpStatsDeltaToKV.
+	h.mu.Lock()
+	is := h.mu.ctx.GetDomainInfoSchema().(infoschema.InfoSchema)
+	h.mu.Unlock()
```

Comment:
Can we wrap these lines in a function like:
```
func (h *Handle) GetInfoSchema() is {
h.mu.Lock()
defer h.mu.Lock()
return h.GetIS().(IS)
```

Reply:
Fixed.